shell/src/main/java/alluxio/cli/bundler/command/CollectLogCommand.java

```diff
@@ -53,15 +157,144 @@ public boolean hasSubCommand() {
   public int run(CommandLine cl) throws AlluxioException, IOException {
     // Determine the working dir path
     mWorkingDirPath = getWorkingDirectory(cl);
-    String logDir = mFsContext.getClusterConf().get(PropertyKey.LOGS_DIR);
 
     // TODO(jiacheng): phase 2 Copy intelligently find security risks
-    // TODO(jiacheng): phase 2 components option
-    FileUtils.copyDirectory(new File(logDir), new File(mWorkingDirPath), true);
+    mIncludedPrefix = new HashSet<>(FILE_NAMES);
+    // Define include list and exclude list
+    if (cl.hasOption(INCLUDE_OPTION_NAME)) {
+      Set<String> toInclude = parseFileNames(cl.getOptionValue(INCLUDE_OPTION_NAME));
+      System.out.format("Include the following filename prefixes: %s%n", toInclude);
+      mIncludedPrefix.addAll(toInclude);
```

Comment:
`mIncludedPrefix` starts off with the entire list. Does that mean it can never be reduced? For example, if I use the option `--include master.log` this will still retrieve all logs because `mIncludedPrefix = new HashSet<>(FILE_NAMES)`?

Reply:
Great question. This `--include-logs` was designed for extra logs that we don't know, like "alluxio_gc.log" which some people use for gc logs. These logs should be collected additional to all the rest of logs.

The use case you are thinking is also valid, where people just want a subset, like "master.log" and "master.out". This was not in the scope.

I think I will change the current design to:
`--include-logs <file-names>` to replace the included list. Eg. `--include-logs "master.log, master.out"` copies only master logs and master outs.
`--additional-logs <file-names>` adds sth extra to the include list. Eg. `--additional-logs "alluxio_gc.log"` takes the specified gc logs tgt with all the rest of logs.

Reply:
This is added.